modules/subchannel/test/tests/problems/SFR/sodium-19pin/tests

```diff
@@ -2,60 +2,44 @@
   design = 'TriSubChannel1PhaseProblem.md'
   issues = '#29194 #29490 #29505'
   [test_explicit]
-    type = Exodiff
+    type = CSVDiff
     input = test19_explicit.i
-    exodiff = test19_explicit_out.e
-    custom_cmp = 'test.exodiff'
-    capabilities = 'method!=dbg'
+    csvdiff = test19_explicit_out.csv
     valgrind = NONE
     recover = false
     allow_warnings = true
     max_parallel = 1
     requirement = 'The system will examine the subchannel explicit solver for a 19pin liquid sodium cooled assembly'
   []
   [test_implicit]
-    type = Exodiff
+    type = CSVDiff
     input = test19_implicit.i
-    exodiff = test19_implicit_out.e
-    custom_cmp = 'test.exodiff'
-    capabilities = 'method!=dbg'
+    csvdiff = test19_implicit_out.csv
     valgrind = NONE
     recover = false
     allow_warnings = true
     max_parallel = 1
     requirement = 'The system will examine the subchannel implicit solver for a 19pin liquid sodium cooled assembly'
   []
   [test_monolithic]
-    type = Exodiff
+    type = CSVDiff
     input = test19_monolithic.i
-    exodiff = test19_monolithic_out.e
-    custom_cmp = 'test.exodiff'
-    capabilities = 'method!=dbg'
+    csvdiff = test19_monolithic_out.csv
     abs_zero = 1e-7
     valgrind = NONE
     recover = false
     max_parallel = 1
     requirement = 'The system will examine the subchannel monolithic solver for a 19pin liquid sodium cooled assembly'
   []
   [test_full_monolithic]
-    type = Exodiff
+    type = CSVDiff
     input = test19_full_monolithic.i
-    exodiff = test19_full_monolithic_out.e
-    custom_cmp = 'test.exodiff'
-    capabilities = 'method!=dbg'
+    csvdiff = test19_full_monolithic_out.csv
+    rel_err = 5e-4
     abs_zero = 1e-7
```

Comment:
are these differences between civet and your machine? If so add a comment.
If not, please regold to get the same tol as the others

Reply:
talked in person. Changes in validation folders meet this idea already